guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -3,6 +3,8 @@
 *********************************************
 Installing SQream with Monit
 *********************************************
+Monit is an Open Source utility for managing and monitoring Unix systems and conducting automatic maintenance and repair. For example, it can be used for executing meaningful causal actions in error situations, such as starting processes that are not running.
```

Comment:
I'd propose: Monit is a free, open-source process supervision tool for Unix and Linux. With Monit, system status can be viewed directly from the command line, or via the native HTTP web server. Monit is able to do automatic maintenance, repair, and run meaningful causal actions in error situations. (taken from Wikipedia)